srcpkgs/orientdb/template

```diff
@@ -1,18 +1,17 @@
 # Template file for 'orientdb'
 pkgname=orientdb
-version=2.2.29
-revision=2
+version=3.1.11
+revision=1
 archs="i686 x86_64"
 conf_files="/etc/${pkgname}/*"
-wrksrc="${pkgname}-community-${version}"
 depends="virtual?java-runtime"
 short_desc="Distributed Graph Database"
 maintainer="bougyman <bougyman@rubyists.com>"
 license="Apache-2.0"
+homepage="https://orientdb.org"
+distfiles="https://s3.us-east-2.amazonaws.com/orientdb3/releases/${version}/orientdb-${version}.tar.gz"
+checksum=73830279a5587de17ea4f5864598a0829bb03212d6cd36d185fe126a46086236
 patch_args="-p1"
```

Comment:
When you're moving this (see xlint test result), please also use `-Np1`

Reply:
Well, now my comment is outdated :p

Reply:
Ugh, saw this 30 seconds too late...